### ISSUE TITLE
Fetal intra-abdominal umbilical vein varix

### ISSUE THREAD

**Issue by patt1morgan** [obophenotype/human-phenotype-ontology#7018]
**Preferred term label:** Fetal intra-abdominal umbilical vein varix

**Synonyms**  FIUVV, intra- abdominal umbilical vein varix

**Definition (free text, please give PubMed ID)**
FIUVV is defined by the focal dilatation of the fetal umbilical vein between its entry in the abdomen and its ending in the portal system. The dilatation can be defined using one of the following criteria: a portion of intra-abdominal umbilical vein that is at least 50% wider than the non-dilated portion; a dilatation of ≥ 9 mm; or a dilatation greater than 2 SD above the mean value for gestational age.       PMID: 28876490  PMID: 24883288

**Parent term (use hpo.jax.org/app)**
Umbilical vein varix HP:0030656

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)** FIUVV is associated with fetal aneuploidy (trisomy 9, 18, and 21) (up to 6.3%) and multiple malformations (up to 35.3%)  PMID 30090795,PMID: 24631035

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@jokemuys, @rhiannonmellis -  Greetings!  this is another of the new term requests for expanding the HPO.  We hope you will review this and approve it as it is, or add any input you feel will improve the definition.  Thank you in advance for taking the time to help us.

**Comment by rhiannonmellis** [comment]
Hi! This looks great to me. Just one suggestions: would it be helpful to also add "fetal umbilical vein dilatation" as another synonym to make it easier to find in the search function for a non-expert?

**Comment by pnrobinson** [comment]
@patt1morgan , @rhiannonmellis thanks, I have added the term and the synonym as suggested
